### PR TITLE
OSSMDOC-626: Check known issues file for fixed issues.

### PR DESCRIPTION
**Pull request by JStickler**
Version(s):  OCP 4.6 - 4.11

Issue: [OSSMDOC-626](https://issues.redhat.com/browse/OSSMDOC-626)

Link to docs preview:
http://file.bos.redhat.com/jstickle/OSSMDOC-626/service_mesh/v2x/servicemesh-release-notes.html

This PR 
* Checks the Known Issues module for issues that have been fixed and moves them to the Fixed Issues module
* Adds comments for issues that have been closed in JIRA because they were documented (there were some questions about why some of these were still in the release notes if they were closed)
* Rearranges some of the issues in numerical order for easier reference/ findability

QE review: pbajjuri20
Peer review: nalhadef

### CODE REVIEW

**Comment by JStickler** [comment]
Woke up this morning and realized that if the comments get separated from their issue bullets when adding/removing issues, it could be confusing.  Added the issue numbers to the "Keep in RN" comments for clarity.

File: modules/ossm-rn-fixed-issues.adoc [hunk 1]
```diff
@@ -49,6 +49,15 @@ Namespace starting with `kube` is hidden from Kiali.
```
Comment (nalhadef) [inline modules/ossm-rn-fixed-issues.adoc:49]: I didn't see a content type at the beginning of this file.

File: modules/ossm-rn-fixed-issues.adoc [hunk 2]
```diff
@@ -49,6 +49,15 @@ Namespace starting with `kube` is hidden from Kiali.
 
 * link:https://issues.redhat.com/browse/OSSM-285[OSSM-285] When trying to access the Kiali console, receive the following error message "Error trying to get OAuth Metadata". The workaround is to restart the Kiali pod.
 
+* link:https://issues.redhat.com/browse/MAISTRA-2735[MAISTRA-2735] The resources that the Service Mesh Operator deletes when reconciling the SMCP changed in {SMProductName} version 2.1. Previously, the Operator deleted a resource with the following labels:
+
+** `maistra.io/owner`
+** `app.kubernetes.io/version`
+
++
+Now, the Operator ignores resources that don't also include the `app.kubernetes.io/managed-by=maistra-istio-operator` label. If you create your own resources, you should not add the `app.kubernetes.io/managed-by=maistra-istio-operator` label to them.
```
Comment (nalhadef) [inline modules/ossm-rn-fixed-issues.adoc:58]: Change "don't" to "does not."

File: modules/ossm-rn-fixed-issues.adoc [hunk 3]
```diff
@@ -88,9 +99,11 @@ Upgrading the operator to 2.0 might break client tools that read the SMCP status
...
+
 * link:https://issues.redhat.com/browse/MAISTRA-1983[MAISTRA-1983] _Migration to 2.0_ Upgrading to 2.0.0 with an existing invalid `ServiceMeshControlPlane` cannot easily be repaired. The invalid items in the `ServiceMeshControlPlane` resource caused an unrecoverable error. The fix makes the errors recoverable. You can delete the invalid resource and replace it with a new one or edit the resource to fix the errors. For more information about editing your resource, see [Configuring the Red Hat OpenShift Service Mesh installation].
 
-* link:https://issues.redhat.com/browse/MAISTRA-1502[Maistra-1502] As a result of CVEs fixes in version 1.0.10, the Istio dashboards are not available from the *Home Dashboard* menu in Grafana. The Istio dashboards still exist. To access them, click the *Dashboard* menu in the navigation panel and select the *Manage* tab.
+* link:https://issues.redhat.com/browse/MAISTRA-1502[MAISTRA-1502] As a result of CVEs fixes in version 1.0.10, the Istio dashboards are not available from the *Home Dashboard* menu in Grafana. The Istio dashboards still exist. To access them, click the *Dashboard* menu in the navigation panel and select the *Manage* tab.
```
Comment (nalhadef) [inline modules/ossm-rn-fixed-issues.adoc:106]: Delete "The Istio dashboards still exist."

Comment (pbajjuri20) [reply modules/ossm-rn-fixed-issues.adoc:106]: https://issues.redhat.com/browse/MAISTRA-1502, This issue is marked as Closed

File: modules/ossm-rn-fixed-issues.adoc (hunk 1 above)
Comment (JStickler) [reply modules/ossm-rn-fixed-issues.adoc:49]: Add to both files.

**Pull request merged**